tools/purge_duplicate_rubyrep_triggers.rb

```diff
@@ -0,0 +1,47 @@
+# This script removes triggers that could cause issues with rubyrep.
+# When tables are renamed, the rubyrep trigger should be removed and recreated.
+# When they are not, it is possible to end up with multiple triggers on the renamed table.
+# Both triggers will run, but the old one will insert pending changes for a table that no longer exists.
+# When this happens with high-churn tables it can cause the replicate process to timeout.
+
+require 'pg'
+
+TRIGGER_QUERY = <<-SQL.freeze
+SELECT relname, array_agg(tgname) AS triggers
+FROM
+  pg_trigger t JOIN
+  pg_class c ON t.tgrelid = c.oid
+WHERE
+  t.tgname like 'rr%'
+GROUP BY relname
+SQL
+
+def sql_array_to_ruby(sql_arr)
+  sql_arr[1..-2].split(",")
```

Comment:
Maybe put an example of sql_arr is at this point in a comment since 1..-2 isn't immediately obvious... at least for me.